tokio/src/runtime/metrics/runtime.rs

```diff
@@ -88,6 +88,25 @@ impl RuntimeMetrics {
         self.handle.inner.active_tasks_count()
     }
 
+    /// Returns the total number of tasks spawned in this runtime.
+    ///
```

Comment:
It would make sense to mention in the documentation how this differs from `active_tasks_count`.

Reply:
Count is consistent with https://prometheus.io/docs/concepts/metric_types/#counter which is monotonically increasing. I would instead recommend renaming `active_tasks_count` to be `num_active_tasks`. @Darksonn are you comfortable with a breaking change here as this is still unstable API?

Reply:
Breaking changes are okay in unstable features, but in this case we can reduce the impact of the breaking change.

Please duplicate the method and mark the old name with `#[deprecated]`. Then file a bug to remove the old name in a later release.